setup.py

```diff
@@ -57,9 +57,9 @@ def main():
             'requests>=2.12.0,==2.*',
             'jsonschema>=2.5.1,==2.*',
             'six>=1.10,<=1.99',
-            'joblib>=0.11,<=0.11.99',
+            'joblib>=0.11,<=0.13.99',
```

Comment:
why not 0.14, out of curiosity?

Reply:
It's not released yet, so I can't test on it.

Reply:
oh, sorry, i'd only looked at https://joblib.readthedocs.io/en/latest/developing.html#latest-changes, which makes it look like it's released.

Reply:
That confused me too, but there's no tag for it yet.